src/main/java/TaskList.java

```diff
@@ -0,0 +1,11 @@
+import task.Task;
+
+import java.util.ArrayList;
+
+public class TaskList extends ArrayList<Task> {
+
+    public TaskList() {
+        super();
+    }
+
+}
```

Comment:
Interesting implementation, but could this potentially expose the TaskList to other users if they recognise that the method calls are identical to the ArrayList class? Users could modify the TaskList directly... 😱